vendordeps/PathplannerLib.json

```diff
@@ -1,7 +1,7 @@
 {
     "fileName": "PathplannerLib.json",
     "name": "PathplannerLib",
-    "version": "2023.2.3",
+    "version": "2023.3.1",
```

Comment:
The version needs to be upgraded again. Seems to be why the CI workflow is breaking